Update the autocapture config docs (#6755)
PostHog/posthog.com: contents/docs/libraries/js/index.mdx

| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------ |
| `api_host`<br/><br/>**Type:** String<br/>**Default:** `https://app.posthog.com`                                                                                                           | URL of your PostHog instance.                                                                                                  |
| `ui_host`<br/><br/>**Type:** String<br/>**Default:** undefined                                                                                                                            | If using a [reverse proxy](/docs/advanced/proxy) for `api_host` then this should be the actual PostHog app URL (e.g. app.posthog.com) |

> | `autocapture`<br/><br/>**Type:** Boolean | AutocaptureConfig<br/>**Default:** `true`                                                                                                                          | Determines if PostHog should [autocapture](#autocapture) events. This setting does not affect capturing pageview events (see `capture_pageview`).  See below for `AutocaptureConfig`          |

Author: ```suggestion
| `autocapture`<br/><br/>**Type:** Boolean or AutocaptureConfig<br/>**Default:** `true`                                                                                                                          | Determines if PostHog should [autocapture](#autocapture) events. This setting does not affect capturing pageview events (see `capture_pageview`).  [See below for `AutocaptureConfig`](#autocaptureconfig))          |
```

Any custom event capturing (`posthog.capture`), `$identify`, `$set`, `$set_once` and basically any other calls not detailed above will work as expected when `/decide` is disabled.

> ## AutocaptureConfig

Reviewer: For some reason this doesn't work as is 🤷 
```suggestion
## AutocaptureConfig
```

> `autocapture` config can take an object providing further config to autocapture's behavior.

Reviewer: ```suggestion
The `autocapture` config takes an object providing full control of autocapture's behavior.
```

| Attribute                                                                                                                                                                                 | Description                                                                                                                    |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------ |

> | `url_allowlist`<br/><br/>**Type:** Array of String or Regexp<br/>**Default:** `undefined`                                                                                                          | List of URLs to allow autocapture on, can be strings to match or regexes e.g. ['https://example.com', 'test.com/.*']. NB an empty list means no URLs are allowed for capture.                                                                                                 |

Reviewer: ```suggestion
| `url_allowlist`<br/><br/>**Type:** Array of Strings or Regexp<br/>**Default:** `undefined`                                                                                                          | List of URLs to enable autocapture on, can be string or regex matches e.g. `['https://example.com', 'test.com/.*']`. An empty list means no URLs are allowed for capture, `undefined` means all URLs are.                                                                                                 |
```

> | `dom_event_allowlist`<br/><br/>**Type:** `'click' or 'change' or 'submit'`<br/>**Default:** `undefined`                                                                                                          | List of DOM events to allow autocapture on.          NB an empty list means no events are allowed for capture.                                                                                            |

Reviewer: ```suggestion
| `dom_event_allowlist`<br/><br/>**Type:** Array of Strings <br/>**Default:** `undefined`                                                                                                          | An array of DOM events, like 'click', 'change', 'submit',  to enable autocapture on. An empty array means no events are enable for capture, `undefined` means all are.                                                                                            |
```

> | `element_allowlist`<br/><br/>**Type:** `'a' or 'button' or 'form' or 'input' or 'select' or 'textarea' or 'label'`<br/>**Default:** `undefined`                                                                                                          | List of DOM elements to allow autocapture on.          NB an empty list means no elements are allowed for capture.                                                                                            |

Reviewer: ```suggestion
| `element_allowlist`<br/><br/>**Type:** Array of Strings <br/>**Default:** `undefined`                                                                                                          | An array of DOM elements, like 'a', 'button', 'form', 'input', 'select', 'textarea', or 'label', to allow autocapture on. An empty array means no elements are enabled for capture, `undefined` means all elements are enabled.                                                                                            |
```

> | `css_selector_allowlist`<br/><br/>**Type:** `String[]`<br/>**Default:** `undefined`                                                                                                          | List of CSS selectors to allow autocapture on.          NB an empty list means no css selectors are allowed for capture.                                                                                            |

Reviewer: ```suggestion
| `css_selector_allowlist`<br/><br/>**Type:** Array of Strings <br/>**Default:** `undefined`                                                                                                          | An array of CSS selectors to enable autocapture on. An empty array means no CSS selectors are allowed for capture, `undefined` means all CSS selectors are.                                                                                            |
```

> | `element_attribute_ignorelist`<br/><br/>**Type:** `String[]`<br/>**Default:** `undefined`                                                                                                          | List of element attributes that autocapture will not capture.          NB an empty list means no attributes are ignored.                                                                                     |

Reviewer: Clarification needed here.
```suggestion
| `element_attribute_ignorelist`<br/><br/>**Type:** Array of Strings <br/>**Default:** `undefined`                                                                                                          | An array of element attributes that autocapture will not capture. An empty array means all events are ignored, `undefined` means elements with any attribute are captured.                                                                                     |
```

Author: ```suggestion
| `element_attribute_ignorelist`<br/><br/>**Type:** Array of Strings <br/>**Default:** `undefined` |  An array of element attributes that autocapture will not capture. Both an empty array and `undefined` mean any of the attributes from the element are captured. |
```

Author: This one is slightly different since its an `ignore` value and not an `allow` value. Edited slightly based on your suggestion but obvs feel free to edit further 😊

Reviewer: Ahh, that's good. Thanks!

> 
> 

Reviewer: ```suggestion
```
